file-formats/dobj/README.md

```diff
@@ -2,4 +2,4 @@
 
 File | Cardinality | Definition | Schema | Example
 :--- | :---  | :--- | :--- | :---
-`<name>.dobj.json` | 1 | [`zif_aff_dobj_v1.intf.abap`](./type/zif_aff_dobj_v1.intf.abap) | [`dobj-v1.json`](./dobj-v1.json) | [`z_aff_example_dobj.dobj.json`](./examples/z_aff_example_dobj.dobj.json)
+`<name>.dobj.json` | 1 | [`zif_aff_dobj_v1.intf.abap`](./type/zif_aff_dobj_v1.intf.abap) | [`dobj-v1.json`](./dobj-v1.json) | [`zbc_sflight_destruction.dobj.json`](./examples/zbc_sflight_destruction.dobj.json)
```

Comment:
Did you change the example object by intention? It would be great if we could keep the name `z_aff_examle_dobj` in this repository? 

The new content of the object example be kept.

Reply:
I changed it intentionally to set it as bc_sflight_destruction as this sample object is delivered and available in systems. will change it back to z_aff_example_dobj if bc_sflight_destruction is not needed

Reply:
> I changed it intentionally to set it as bc_sflight_destruction as this sample object is delivered and available in systems. will change it back to z_aff_example_dobj if bc_sflight_destruction is not needed

Please remove `file-formats/dobj/examples/zbc_sflight_destruction.dobj.json` from this change as well.
BTW: commit with message like `Add files to ABAPLINT` is really misleading and will confuse peple when reading the git history. ABAPLINT is a static code checker. Just write what you really did. E.g. `remove zbc_sflight_destruction file`

Reply:
Have removed the file and the latest changes uploaded

Reply:
The build still fails with the same error